server/test/auth/CiviFormProfileMergerTest.java

```diff
@@ -0,0 +1,134 @@
+package auth;
+
+import static java.util.concurrent.CompletableFuture.completedFuture;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+import java.util.Collections;
+import java.util.Optional;
+import models.Account;
+import models.Applicant;
+import org.junit.Before;
+import org.junit.Test;
+import org.pac4j.core.profile.CommonProfile;
+import org.pac4j.core.profile.UserProfile;
+import org.pac4j.oidc.profile.OidcProfile;
+import repository.ResetPostgres;
+import repository.UserRepository;
+
+public class CiviFormProfileMergerTest extends ResetPostgres {
+
+  private static final String EMAIL_ATTR = "user_emailid";
+  private static final String EMAIL1 = "foo@bar.com";
+  private static final String EMAIL2 = "bar@foo.com";
+  private static final Long ACCOUNT_ID = 12345L;
+  private static final UserProfile USER_PROFILE = new CommonProfile();
```

Comment:
Classes tend to not be deeply immutable, so they shouldn't use const case per the style guide.  EG: they can effectively change and CONSTS should never change.

We likely want to make a new one for each test just in case too.